config/initializers/roles_db.rb

```diff
@@ -13,3 +13,12 @@
     config.strategy_class = "RolesDb::LocalRoles"
   end
 end
+
+RolesDb.configure do |config|
+  config.ssl_cert_key_file = Rails.root.join("config", "certs", "outcomes-key.pem")
+  config.ssl_cert_file = Rails.root.join("config", "certs", "outcomes.cer")
+  config.ssl_ca_cert_file = Rails.root.join("config", "certs", "mit.pem")
+  config.roles_wsdl_file = Rails.root.join("config", "certs", "roles.wsdl")
+  config.ssl_cert_key_password = "abetiscool"
```

Comment:
I don't think so. I was going to follow up with Jon about scrubbing it. But for now, it's just moved.